### PR TITLE
[PRM-841] - Code cleanup

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by ruthdas03** [comment]
Please resolve the conflicts.

**Comment by LewisMitchell** [comment]
Done

File: app/controllers/ComplianceController.scala [hunk 1]
```diff
@@ -20,33 +20,16 @@ import play.api.libs.json.Json
...
-      }
-    }
-  }
-
-  def getComplianceDataFromDES(vrn: String, fromDate: String, toDate: String): Action[AnyContent] = Action.async {
-    implicit request => {
-      complianceService.getComplianceDataFromDES(vrn, fromDate, toDate).map {
+      complianceService.getComplianceData(vrn, fromDate, toDate).map {
         _.fold(
           error => Status(error),
```
Comment (ruthdas03) [inline app/controllers/ComplianceController.scala:34]: Can we add logging here in case of error.

**Review by ruthdas03 — COMMENTED** [review]
The clean up is ok however in the nomenclature where 'DES' is removed it signifies that from 'compliance' there's always a generic implementation. It's ok as long as we don't have to handle any other calls/implemenations from 'compliance'.

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**